libs/feature/editor/src/lib/components/record-form/form-field/form-field-overviews/form-field-overviews.component.ts

```diff
@@ -68,7 +88,7 @@ export class FormFieldOverviewsComponent {
             })
           }
         },
-        error: this.errorHandle,
+        error: (error: any) => this.handleError(error.error.message),
```

Comment:
Same remark as above: handleError should take an Error as input.

Also please keep in mind that every `any` in the code is technical debt, and we should strive to avoid & get rid of them as much as possible.